packages/textfield/README.md

```diff
@@ -0,0 +1,267 @@
+# `<mwc-textfield>` & `<mwc-textarea>` [![Published on npm](https://img.shields.io/npm/v/@material/mwc-textfield.svg)](https://www.npmjs.com/package/@material/mwc-textfield)
+
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
+
+Textfields let users enter and edit text.
+
+<img src="images/action.png" width="28.5714%">
+
+[Material Design Guidelines: textfields](https://material.io/design/components/text-fields.html)
+
+## Installation
+
+```sh
+npm install @material/mwc-textfield
+```
+
+> NOTE: The Material Web Components are distributed as ES2017 JavaScript
+> Modules, and use the Custom Elements API. They are compatible with all modern
+> browsers including Chrome, Firefox, Safari, Edge, and IE11, but an additional
+> tooling step is required to resolve *bare module specifiers*, as well as
+> transpilation and polyfills for Edge and IE11. See
+> [here](https://github.com/material-components/material-components-web-components#quick-start)
+> for detailed instructions.
+
+## Example usage
+
+### Standard / Filled
+
+<img src="images/standard.png" width="28.5714%">
+
+```html
+<mwc-textfield label="My Textfield"></mwc-textfield>
+
+<script type="module">
+  import '@material/mwc-textfield';
+</script>
+```
+
+### Icon - Leading
+
+<img src="images/icon-leading.png" width="28.5714%">
+
+```html
+<mwc-textfield label="My Textfield" icon="event"></mwc-textfield>
+
+<script type="module">
+  import '@material/mwc-textfield';
+  import '@material/mwc-icon/mwc-icon-font.js';
+</script>
+```
+
+### Icon - Trailing
+
+<img src="images/icon-trailing.png" width="28.5714%">
```

Comment:
Kind of a nit, but the spec uses the `cancel` icon instead of `delete` as its example for a clear icon (https://material.io/design/components/text-fields.html#anatomy)

Reply:
I used delete to mirror their mdc demos:

https://material-components.github.io/material-components-web-catalog/#/component/text-field